WorldsAdriftRebornGameServer/Game/ItemHelper.cs

```diff
@@ -0,0 +1,166 @@
+using System.Diagnostics.CodeAnalysis;
+using Bossa.Travellers.Inventory;
+using Improbable.Collections;
+
+namespace WorldsAdriftRebornGameServer.Game
+{
+    public static class ItemHelper
+    {
+        private static Dictionary<string, ValidItem> _allItems = new Dictionary<string, ValidItem>();
+        private const string ItemPath = @"C:\Users\CHANGEME\WorldsAdriftReborn\WorldsAdriftRebornGameServer\Game\itemData.json";  
```

Comment:
maybe put this file next to the game server .exe (or a folder next to it for potentially more config files) and then load it from a relative path?